contracts/staking/PoSValidatorManager.sol

```diff
@@ -120,18 +173,34 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         return uptime;
     }
 
-    function _processStake(uint256 stakeAmount) internal virtual returns (uint64) {
+    function _initializeValidatorRegistration(
+        ValidatorRegistrationInput calldata registrationInput,
+        PoSValidatorRequirements calldata requirements,
+        uint256 stakeAmount
+    ) internal virtual returns (bytes32) {
         PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+        // Validate and save the validator requirements
+        require(
+            requirements.minStakeDuration >= $._minimumStakeDuration,
+            "PoSValidatorManager: invalid min stake duration"
+        );
```

Comment:
I don't know if we should have to check this. I think the validator settings it's own `minStakeDuration` to 0 should be valid if they don't care, we will just still enforce `_minimumStakeDuration` anyway.

Reply:
I think validators should know the `minStakeDuration` they're required before their funds get locked. Wouldn't want cases where validators dont know of the min stake duration then get caught off guard when initializing end, and probably perform badly if they aren't happy with the stake duration needed.